lib/interactivity-api.php

```diff
@@ -16,14 +16,14 @@ function gutenberg_reregister_interactivity_script_modules() {
 
 	wp_register_script_module(
 		'@wordpress/interactivity',
-		gutenberg_url( '/build/interactivity/' . ( SCRIPT_DEBUG ? 'debug.min.js' : 'index.min.js' ) ),
+		gutenberg_url( '/build/script-modules/interactivity/' . ( SCRIPT_DEBUG ? 'debug.min.js' : 'index.min.js' ) ),
```

Comment:
How about fully mirroring `build` folder inside top-level `build-module`?

```suggestion
		gutenberg_url( '/build-module/interactivity/' . ( SCRIPT_DEBUG ? 'debug.min.js' : 'index.min.js' ) ),
```

What cons do I miss?

Reply:
I'm not sure, maybe there aren't problems 🤷 

I didn't think too much about this. I kept it in the same place but renamed from `interactivity` to `script-modules` but that's something I'm happy to explore.

Reply:
This way we could keep two top-level
build folders mirroring also how it works for individual packages.

Reply:
I've used that approach 👍